contribs/gmf/apps/desktop_alt/sass/desktop_alt.scss

```diff
@@ -14,3 +13,53 @@ div .ngeo-displaywindow {
   right: initial;
   top: $topbar-height + $app-margin + 3 * $map-tools-size;
 }
+
+#gmf-map-swipe {
+  position: absolute;
+  z-index: 2;
+  width: 100%;
+}
+
+ngeo-mapswipe .swipe-input {
+  background: none;
+  height: 0;
+  opacity: 0.7;
+  outline: none;
+  pointer-events: none;
+  transition: opacity 0.2s;
+  -webkit-appearance: none;
+  -webkit-transition: 0.2s;
+  width: 100%;
+
+  &::-webkit-slider-runnable-track {
+    background: none;
+    height: none;
+  }
+
+  &::-webkit-slider-thumb {
+    appearance: none;
+    border: 1px solid #7b8e7c;
+    cursor: pointer;
+    height: 900px;
+    pointer-events: auto;
+    -webkit-appearance: none;
+    width: 25px;
+    background: url("../image/right-and-left-arrows-square-button.png") #849e8b;
+    background-size: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    padding-left: 10px;
+    margin-left: -10px;
+  }
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+ngeo-mapswipe div {
+  vertical-align: middle;
+  width: 100%;
+  position: absolute;
+  top: 0;
+}
```

Comment:
This styles should be in a separate file... (like this it's impossible to include it in the desktop)